HIP/hip-583.md

```diff
@@ -10,220 +10,518 @@ status: Accepted
+| Any other alias - ED25519, mutlisig, complex etc. | associated `account-num alias`       | True    |
 
-The Mirror Node can already ingest account creation transactions with alias values.
-This will only increate the applicable parent trnasactions that can cause this flow.
-Where it was limited to CryptoTransfer it will now include CryptoCreate
 
 ### Wallet/Exchange Implications
 
-With the suggested changes, the alias property of an account will become more applicable and will need greater visibility.
+With the suggested changes, the alias property of an account will become more applicable and will need greater
+visibility.
 
-To provide greater web3 user scenario support Wallets & Exchanges should
+To provide greater web3 user scenario support Wallets & Exchanges should:
 
 - Display the account alias property of an account
 - Create an ECDSA based account with alias when full smart contract support is desired
-- Accept the use of an alias as a account identifier in HBAR and token transfers
+- Accept the use of an alias as an account identifier in HBAR and token transfers
 - Accept the use of an alias during account creation
 - Add support to check if an account exists with a given alias
 
 ### Performance Implications
 
-The mass adoption of alias setting on new and existing accounts will have minor but notable memory implications on the Hedera ecosystem
+`CryptoCreate` transactions will see extra bytes for every account created with an alias.
 
-`CryptoCreate` transactions will see extra bytes for every account created with an alias
+The size of the bytes in each case depends on the governing account key type. For an `ECDSA` based account the alias
+will always be a valid Ethereum address and will be 20 bytes long. In the `ED25519` case the alias will be a valid
+protobuf serialized public key of 35 or 68 bytes depending on encoding.
 
-The size of the bytes in each case depends on the governing account key type. For an `ECDSA` based account the alias will always be a valid Ethereum address and will be 20 bytes long. In the `ED25519` case the alias will be a valid protobuf serialized public key of 35 or 68 bytes depending on encoding.
+Overall, we anticipate no significant performance impact from this feature.
 
-## Security Implications
+Security Implications
+------------------------------------------------------------------------------------------------------------------------
 
-The `lazy-create` account flow requires additional checks be added to ensure transactions involving an account with only a `public-address` are not processed without a matching `public-key`. 
+The `hollow-create` account flow requires additional checks be added to ensure transactions involving an account with
+only an `alias` and no key (i.e. Hollow accounts) are not processed. The `CryptoCreate` flow must verify the transaction
+is signed by the key represented by the alias (both `key alias` and `evm address alias`).
```

Comment:
Here we should have only the `evm address alias` left and remove the `key alias`.